pkg/types/ccipocr3/generic_types.go

```diff
@@ -108,7 +116,15 @@ func (c CCIPMsg) String() string {
 }
 
 type CCIPMsgBaseDetails struct {
-	ID          Bytes32       `json:"id"`
+	// ID is a unique identifier for the message, it should be unique across all chains.
+	ID string `json:"id"`
```

Comment:
Just a disambiguating detail
```suggestion
	// ID is a unique identifier for the message, it should be unique across all chains.
	// It is generated on the chain in which the CCIP send is requested (i.e the source chain of a message).
	ID string `json:"id"`
```